src/main.rs

```diff
@@ -60,8 +85,19 @@ fn main() {
                     if c == 0 {
                         break;
                     }
-                    let now = Local::now().format("%T%.3f");
-                    print!("{} {}", format!("[{}]", now).bright_black(), line);
+                    let time_now = Local::now().format("%T%.3f");
+                    if file_writer.is_some() {
+                        match writeln!(file_writer.as_ref().unwrap(), "[{}] {}", &time_now, &line) {
```

Comment:
Is this a good way of doing it? because I read into uwrap and it basically just assumes "some()" value exists and returns it, which it will in this case, because we explicitly check it in the line above?